sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py

```diff
@@ -263,8 +264,13 @@ class AioHttpTransportResponse(AsyncHttpResponse):
     def __init__(self, request: HttpRequest, aiohttp_response: aiohttp.ClientResponse, block_size=None) -> None:
         super(AioHttpTransportResponse, self).__init__(request, aiohttp_response, block_size=block_size)
         # https://aiohttp.readthedocs.io/en/stable/client_reference.html#aiohttp.ClientResponse
+        headers = aiohttp_response.headers
+        if isinstance(headers, CIMultiDictProxy):
```

Comment:
Why????????

Reply:
Seems like I thought it in an over-complicated way.